controllers/api/post-routes.js

```diff
@@ -84,7 +79,7 @@ router.post('/',   (req, res) => {
 });
 
 
-router.put('/:id', withAuth,  (req, res) => {
+router.put('/:id', (req, res) => {
   Post.update(
     {
       title: req.body.title
```

Comment:
I think we should add to edit all fields

Reply:
I think this is fixed now